src/requestHandler.ts

```diff
@@ -1,4 +1,8 @@
 class RequestHandler {
+  backendService: any;
```

Comment:
Transforming this file into TypeScript was necessary to maintain folder structure with grunt-ts compilation. The right types for this file (and all other missing JavaScript files) will be addressed in a separate PR.